homepages/assets/less/single.less

```diff
@@ -219,12 +219,25 @@
     padding-bottom: 0;
     margin-top: 20px;
   }
-  .row-fluid>[class*='span'] {
-    .widget {
-      border: none;
-    }
-    &:last-child .widget {
-      border-right: none;
+  .row-fluid {
+    & > .span1,
+    & > .span2,
+    & > .span3,
+    & > .span4,
+    & > .span5,
+    & > .span6,
+    & > .span7,
+    & > .span8,
+    & > .span9,
+    & > .span10,
+    & > .span11,
+    & > .span12 {
+      .widget {
+        border: none;
+      }
+      &:last-child .widget {
```

Comment:
should this line be inside the `.span12`?
It looks like in the old version it was applied to all of the span's, not just `span12`